php-gd extension is required for Drive setImage

```
$ make test-php-phan
composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Nothing to install, update or remove
Generating autoload files
49 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
vendor/bin/phan --config-file .phan/config.php --require-config-exists
   analyze ████████████████████████████████████████████████████████████ 100.0% 950MB/960MB
src/Drive.php:340 PhanUndeclaredTypeParameter Parameter $image has undeclared type \GdImage
make: *** [Makefile:61: test-php-phan] Error 1
```

After I do `sudo apt install php-gd` then it passes.

But this is currently only needed by:
```
    /**
     * @todo This function is not implemented yet! Place, name and signature of the function might change!
     */
    public function setImage(\GdImage $image): Drive
    {
        // upload image to dav/spaces/<space-id>/.space/<image-name>
        // PATCH space
        throw new NotImplementedException(Ocis::FUNCTION_NOT_IMPLEMENTED_YET_ERROR_MESSAGE);
    }
```
which is not implemented yet anyway.

And README only has:
```
### Required PHP Libraries
- php-bcmath
- php-gmp
```

Should we remove `\GdImage` from the parameter declaration for now?
(and write a comment that `\GdImage` is expected to be the parameter type when implemented)

That will avoid this silly little thing in the current v1 releases.

--- a/src/Drive.php
+++ b/src/Drive.php
@@ -336,8 +336,11 @@ class Drive
 
     /**
      * @todo This function is not implemented yet! Place, name and signature of the function might change!
+     *
+     * The type of $image is likely to be \GdImage when implemented.
+     * That would require the php-gd extension to be installed.
      */
-    public function setImage(\GdImage $image): Drive
+    public function setImage(object $image): Drive
     {
         // upload image to dav/spaces/<space-id>/.space/<image-name>
         // PATCH space

